test/passing/refs.default/exp_grouping.ml.ref

```diff
@@ -468,10 +473,11 @@ let _ =
   end
 
 let _ =
-  lazy begin fun xxxxxxxxxxxxxxxxxxxxxxx
-    yyyyyyyyyyyyyyyyyyy
-    zzzzzzzzzzzzzzzzzzzzzzzz
-  ->
+  lazy begin fun
+         xxxxxxxxxxxxxxxxxxxxxxx
+         yyyyyyyyyyyyyyyyyyy
+         zzzzzzzzzzzzzzzzzzzzzzzz
+       ->
     print_endline xxxxxxxxx;
```

Comment:
I'm a bit surprised that the `->` and the body don't have the same indentation baseline. The `->` seems indented relative to the column where the `begin` keyword starts and not relative to a naturally incrementing indent. 
Is it intended ?

I'd expect the `->` and the argument list to be indented 2 and 4 relative to the body of the fun:
```
  lazy begin fun
        argument list
      ->
    body
  end
```

Reply:
This is not intended. What I would have liked  is the following:

```ocaml
lazy begin fun
    argument list
  ->
    body
  end
```

This seemed like a small issue in a very rare situation though, and it might be difficult to fix because everything is in `pro`, and context does not tell you you were lazy when formatting the function

Reply:
The `pro` includes the `lazy` keyword ? This seems to be the right way to make the formatting you want. The box around `begin ... ->` is unwanted and the forced break after the `fun` was not the right thing to do.